apps/users/services.py

```diff
@@ -0,0 +1,69 @@
+from apps.common.exceptions import BackendException
+from apps.common.services import AsyncCRUDBase
+from apps.users.models import OneTimeToken, Student, User, user_list_view, students_list_view
+from settings import Settings
+
+from translitua import translit
+from random import randint
+import hashlib
+import os
+from datetime import datetime, timedelta
+from fastapi import status as http_status
+from fastapi.security import OAuth2PasswordBearer
+
+
+reusable_oauth = OAuth2PasswordBearer(
+    tokenUrl="/login",
+    scheme_name="JWT"
+)
+
+
+def get_login(data):
```

Comment:
All these methods - business logic, move to handlers.py OR (validators.py / helpers.py / utils.py) - depends on logical context.

Reply:
ok